app/src/main/java/com/arctouch/codechallenge/movie/ui/MovieFragment.kt

```diff
@@ -0,0 +1,61 @@
+package com.arctouch.codechallenge.movie.ui
+
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.arctouch.codechallenge.R
+import com.arctouch.codechallenge.core.di.GlideApp
+import com.arctouch.codechallenge.core.domain.model.Movie
+import com.arctouch.codechallenge.core.extensions.buildBackdropUrl
+import kotlinx.android.synthetic.main.movie_fragment.*
+
+/**
+ * Created by Felipe Valadares on 18/05/2019
+ *
+ * A fragment to show the basic details from a movie.
+ */
+class MovieFragment : Fragment() {
+
+    private var movie: Movie? = null
```

Comment:
This fragment only works with a movie so it shouldn´t be nullable, using lateinit ensure that the fragment gets a valid movie or throw an exception, but i think it will be better if this var doesn´t even exists, it should be on the viewModel of the fragment or activity and the fragment only observes